plugins/aws_sts/test/samson_aws_sts/samson_plugin_test.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+require_relative '../test_helper'
+
+SingleCov.covered! uncovered: 5
+
+describe SamsonAwsSts do
+  let(:stage) { stages(:test_staging) }
+
+  describe '.deploy_env_vars' do
+    it 'returns a hash of additional env variables' do
+      stage.aws_sts_iam_role_arn = "some_arn"
+
+      result = SamsonAwsSts.deploy_env_vars(
+        sts_client: Aws::STS::Client.new(stub_responses: true),
+        deploy: stage.deploys.last
+      )
+
+      result.must_equal(
+        STS_AWS_ACCESS_KEY_ID:     'hidden://accessKeyIdType',
+        STS_AWS_SECRET_ACCESS_KEY: 'hidden://accessKeySecretType',
+        STS_AWS_SESSION_TOKEN:     'hidden://tokenType'
+      )
+    end
+  end
+
+  describe 'for a stage with a blank ARN' do
+    it 'returns an empty hash' do
```

Comment:
can simplify the nesting to:
`it 'returns nothing for a stage with a blank ARN' do`